components/table/table-wrapper.js

```diff
@@ -44,12 +44,14 @@ export const tableStyles = css`
 
 	/* header cells */
 	.d2l-table > thead > tr > th,
+	.d2l-table > * > tr.d2l-table-header > th,
 	.d2l-table > * > tr[header] > th {
```

Comment:
Still going to support the old way, at least until I can go through and fix them all... or possibly forever if that's too much work.